src/core/annotation.js

```diff
@@ -1652,6 +1659,7 @@ class TextWidgetAnnotation extends WidgetAnnotation {
     return {
       id: this.data.id,
       value: this.data.fieldValue,
+      defaultValue: this.data.defaultFieldValue || "",
```

Comment:
Like the other cases below, the `|| ""` part should be removed here. It's optional according to the spec and will be `null` if it's not available, which is consistent with the API.